guides/common/modules/con_defining-your-content-library.adoc

```diff
@@ -0,0 +1,19 @@
+[id="defining-your-content-library_{context}"]
+= Definining your content library
+
+To ensure that your {ProjectServer} can manage software and provide it to your hosts, you must enable repositories and synchronize them.
+
+* Enabling repositories allows you to choose the specific software required for your environment.
+By enabling only the necessary repositories, you avoid downloading unnecessary content.
+Different systems might need different sets of software.
```

Comment:
I feel like this line is alluding to lifecycle management and/or the "repository sets" feature. When reading this from a user perspective, my first question was "how do I give different systems different sets of software?"

Reply:
Some of this is actually covered in the next section. Rather than explain it here as well, let's try to avoid making users ask this question at this moment :) I'm deleting the sentence and will rely on section "Defining content access strategies for hosts" to explain that part of content management.